packages/core/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/utils/urlToFile.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+
+export const urlToFile = async url => {
+  try {
+    const res = await axios.get(url, { responseType: 'blob', timeout: 8000 });
```

Comment:
I think we should use the proxy api provided by the backend here instead like we do in the ML